framework/src/packets/tcp.rs

```diff
@@ -390,6 +390,15 @@ impl<E: IpPacket> Tcp<E> {
         )
     }
 
+    #[inline]
+    pub fn stamp_flow(&mut self, flow: Flow) -> Result<()> {
+        self.envelope_mut().set_src(flow.src_ip())?;
+        self.envelope_mut().set_dst(flow.dst_ip())?;
+        self.set_src_port(flow.src_port());
+        self.set_dst_port(flow.dst_port());
+        Ok(())
+    }
```

Comment:
not sure I like this function here. feels like unlike the changes to flow where it's pretty universal and will be used a lot. this feels very specific to that one example. I'd like not to add to the core api unless it's something we feel will get a lot of use.

Reply:
Yep. I’ll just make specific ones in the nf